src/framework/mlt_types.h

```diff
@@ -97,21 +97,63 @@ typedef enum {
 /** Colorspace definitions */
 
 typedef enum {
-    mlt_colorspace_rgb = 0,   ///< order of coefficients is actually GBR, also IEC 61966-2-1 (sRGB)
-    mlt_colorspace_bt709 = 1, ///< also ITU-R BT1361 / IEC 61966-2-4 xvYCC709 / SMPTE RP177 Annex B
+    mlt_colorspace_rgb = 0, ///< order of coefficients is actually GBR, also IEC 61966-2-1 (sRGB)
```

Comment:
This enum was never used in the past. It only existed as a member in mlt_image. But we never used it. So I feel ok about changing the values. The values were previously set to match FFMpeg. I changed some values to match the hard-coded values that were previously in MLT. The values that were not previously used are unchanged from the FFMpeg values.